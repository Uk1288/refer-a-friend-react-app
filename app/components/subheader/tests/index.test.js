import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import SubHeader from '../index';

describe('<SubHeader />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <SubHeader />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
