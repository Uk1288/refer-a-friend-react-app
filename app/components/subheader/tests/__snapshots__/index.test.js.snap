// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubHeader /> should render and match the snapshot 1`] = `
<h1
  class="subheader__StyledSubHeader-sc-1ur12bv-0 fByRsJ"
>
  <span>
    Invite friends, get discount code
  </span>
</h1>
`;
